cmake/modules/SearchInstalledSoftware.cmake

```diff
@@ -664,6 +664,14 @@ if(mysql)
 endif()
 
 #---Check for Oracle-------------------------------------------------------------------
+# OCCI is compiled with libstdc++ which has a different ABI than libc++.
+# So we better turn of Oracle in this situation to prevent linking errors between
+# ROOT using libc++ and libOracle.so using libstdc++.
+if(oracle AND libcxx)
+    message(WARNING "Oracle not supported when compiling with libc++!")
+    set(oracle OFF CACHE BOOL "" FORCE)
+endif()
+
```

Comment:
You are assuming that oracle will never deliver libraries for libc++. Can we not at this moment disable the oracle option `-Doracle=OFF` for the current state of the externals we use.